src/components/repos/RepositoriesPage.js

```diff
@@ -0,0 +1,73 @@
+import React, { useEffect, useState } from 'react';
+import { Link } from 'react-router-dom';
+import D3Chart from '../chart/D3Chart';
+import API from '../../utils/api';
+
+function RepositoriesPage() {
+  const [repos, setRepos] = useState([]);
+  const [d3Data, setD3Data] = useState({});
+  const [username, setUsername] = useState('davidholyko');
+
+  const getData = async () => {
+    const tempData = {};
+    const { data: repoData } = await API.getReposData({
+      username,
+    });
+
+    const excluded = ['public-apis', 'EaselJS'];
```

Comment:
a small note: this line is just because these repos are forked and I wanted to exclude them from the chart. We will want logic later that would exclude getting repo commit history from forked repos

Reply:
Gotcha, added a card for this on the board